### ISSUE TITLE
Uncaught TypeError: Illegal invocation

### ISSUE THREAD

**Issue by ahmadsholehin** [kutlugsahin/react-smooth-dnd#48]
We're getting a `Uncaught TypeError: Illegal invocation` error when using react-smooth-dnd with `@loadable/component`.

Specifically, we're trying to import a component that uses react-smooth-dnd dynamically. As the page loads, the error is thrown. The full stack trace is as below:

```
loadable.esm.js:269 Uncaught TypeError: Illegal invocation
    at Node.get (VM217 8.js:97747)
    at VM217 8.js:97747
    at VM217 8.js:97747
    at Object../node_modules/smooth-dnd/dist/index.js (VM217 8.js:97747)
    at __webpack_require__ (VM210 app.js:64)
    at VM217 8.js:89812
    at Object../node_modules/react-smooth-dnd/dist/index.js (VM217 8.js:89812)
    at __webpack_require__ (VM210 app.js:64)
    at Module../src/components/Sunspot/index.js (VM218 10.js:13301)
    at __webpack_require__ (VM210 app.js:64)
```

Googling on the issue seems to point towards a loss of context or an undefined global variable after webpack builds.

Has anyone faced similar issues? Any help would be much appreciated. We burned a weekend trying to solve this single issue.

*Update:* Wanted to add a note that the application works when running via webpack-dev-server directly, but throws the error above after webpack builds it.

**Comment by steveu** [comment]
This package was generating `Uncaught TypeError: Illegal invocation` when imported in a JEST environment (for me; CRA).

Adding this to the `setupTests.js` file in CRA fixed this warning.

```
Object.defineProperty(global, "Node", {
  value: { firstElementChild: "firstElementChild" }
});
```
Maybe something similar will work. Is smooth-dnd causing the error https://github.com/kutlugsahin/smooth-dnd/issues/36

**Comment by ahmadsholehin** [comment]
Wanted to follow up on this. 

We solved this issue when we realised that one of the plugins in webpack is actually automatically appending a script tag pointing to the built app.js file in index.html. We also hand-coded the script tag in the index.html. Therefore, app.js is called _twice_.

Removing the hand-coded script tag in our index.html solved our problem. 

Hope this helps anyone with similar issues.
